src/main/java/com/cloudbees/jenkins/plugins/bitbucket/endpoints/BitbucketServerEndpoint.java

```diff
@@ -134,12 +141,32 @@ public void setCallCanMerge(boolean callCanMerge) {
     }
 
     public boolean isCallChanges() {
-        return callChanges;
+        return callChanges && BitbucketServerVersion.VERSION_7.equals(serverVersion);
```

Comment:
The call changes options was introduced due to changes in Bitbucket Server 7 but the API call IMHO also works for Bitbucket Server 6. We are still running Bitbucket Server 6 and currently have both options enabled (call merge / call changes) and do not see any errors in the logs.
With these settings we wouldn't need to change anything once the Bitbucket Server is upgraded to 7.x.

TLDR: I do not think you need to restrict this option to Bitbucket Server 7.

Reply:
@jansohn 
If the API is not in Bitbucket 6.x we do not want try to make this call. I would say we don't want to all the setting to be true when it does nothing.   But it could also be argued that we shouldn't force values in ways that might be surprising.